README.md

```diff
@@ -1,4 +1,6 @@
-# [Bazel](https://bazel.build)
+<a href="https://bazel.build/">
+    <img width="400" src="https://cdn.icon-icons.com/icons2/2699/PNG/512/bazel_logo_icon_167814.png" alt="Bazel logo"> 
```

Comment:
We probably do not want to include a logo from a 3rd-party source here.

Reply:
@meisterT  respected sir , you mean it should be designed by contributor itself ?

Reply:
Sorry, I have missed your response. Probably we should add the SVG (https://en.m.wikipedia.org/wiki/File:Bazel_logo.svg) to this repository and then reference that

Reply:
so mine code is fine or any change required !